### ISSUE TITLE
Create test environment using simulator

### ISSUE THREAD

**Issue by bertybuttface** [dmunozv04/iSponsorBlockTV#135]
**Is your feature request related to a problem? Please describe.**
Testing involves using a real device which is inconvenient and cannot be automated in a simple, robust way.

**Describe the solution you'd like**

1. Container that runs the simulator, provides a code for iSponsorBlockTV (or a whole config file)
2. Container that automates the simulator to run some automated tests.

**Describe alternatives you've considered**
Not automating testing

**Additional context**
None

**Comment by dmunozv04** [comment]
PRs are welcome! :)

**Comment by bertybuttface** [comment]
Yeah I get it is a bit of a stretch goal, we would only need to use the simplest of the following.

Simulators:

| Device             | Status |
|:-------------------|:------:|
| Apple TV           |   [Available](https://developer.apple.com/documentation/xcode/interacting-with-your-app-in-the-tvos-simulator)    |
| Samsung TV (Tizen) |   [Available](https://developer.samsung.com/smarttv/develop/getting-started/using-sdk/tv-emulator.html)    |
| LG TV (WebOS)      |   [Available](https://webostv.developer.lge.com/develop/tools/simulator-installation)    |
| Android TV         |   [Available](https://developer.android.com/training/tv/start/start#run-on-a-virtual-device)    |
| Chromecast         |   ?    |
| Google TV          |   ?    |
| Roku               |   ?    |
| Fire TV            |   ?    |
| CCwGTV             |   ?    |
| Nintendo Switch    |   ?    |
| Xbox One/Series    |   ?    |
| Playstation 4/5    |   ?    |

**Comment by dmunozv04** [comment]
Doing it on a web browser faking a TV user agent would probably be the easiest way to proceed, but it's not something I can devote my time to right now
